src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -88,58 +88,63 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         }
 
         DataImportModel dataImportModel = new DataImportModel();
-        Integer rowOffset = 0;
-
         dataImportModel.id = batchId;
-        dataImportModel.dataImportRows = getDataImportRows(batchId, rowOffset);
+        dataImportModel.dataImportRows = getDataImportRows(batchId, 0);
```

Comment:
How does this change effect infinite scrolling? Is the offset not needed anymore?

Reply:
This shouldn't affect infinite scrolling since the offset was only set in a local variable `Integer offset = 0` in this method.

Reply:
Oh I see this one is only used for the initial call.👍🏽